### PR TITLE
Use latest selenium, use chromedriver in headless mode.

### PR DESCRIPTION
**Pull request by inferno-chromium**

### CODE REVIEW

File: src/requirements.txt [hunk 1]
```diff
@@ -17,4 +17,5 @@ pytz==2018.5
 PyYAML==5.1
 requests==2.21.0
 requests-toolbelt==0.9.1
+selenium==3.141.0
```
Comment (inferno-chromium) [inline src/requirements.txt]: This move allows use to use webdriver for fuzzing as well, have something in plan.

File: docker/ci/Dockerfile [hunk 2]
```diff
@@ -38,9 +38,10 @@ RUN echo "deb [arch=amd64] http://storage.googleapis.com/bazel-apt stable jdk1.8
 
 RUN npm install -g bower polymer-bundler
 
-# RUN curl -s https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - && \
-#     echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list && \
-#     apt-get update && apt-get install -y google-chrome-stable
+# Install latest Chrome stable, needed for chromedriver testing.
```
Comment (inferno-chromium) [inline docker/ci/Dockerfile]: This move is to try reenabling js_unittest next in CI. we need to have google chrome stable there first.

File: docker/ci/requirements.txt [hunk 3]
```diff
@@ -15,7 +15,6 @@ pyfakefs==3.4.3
 pylint==1.9.3
 python-mimeparse==1.5.2
 PyYAML==5.1
```
Comment (Dor1s) [inline docker/ci/requirements.txt]: nit: do we need PyYAML in both places, here and in src/requirements.txt ?

Comment (inferno-chromium) [reply docker/ci/requirements.txt]: I think this is needed for lint run
src/local/butler/lint.py:import yaml
It was breaking things, that is why explicitly got added in https://github.com/google/clusterfuzz/pull/336

Comment (inferno-chromium) [reply docker/ci/requirements.txt]: Found it, yes butler lint runs before setup(setup runs butler bootstrap). and lint needs pyyaml
https://github.com/google/clusterfuzz/blob/master/cloudbuild.yaml#L31

Comment (Dor1s) [reply docker/ci/requirements.txt]: Got it, thanks!

**Pull request merged**